.idea/LightGBM.iml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't include `.idea`.
The directory isn't needed for LightGBM, but for IntelliJ IDEA.

Reply:
Oh... these files was included mistakenly. So deleted. Could you re-check? Thank you!